cli/compose/loader/volume.go

```diff
@@ -112,6 +112,11 @@ func isFilePath(source string) bool {
 		return true
 	}
 
+	// windows named pipes
+	if strings.HasPrefix(source, `\\`) {
```

Comment:
I wonder if we can make this slash-insensitive (like case-insensitive, but for slashes ;-) ).
From a Mac/Linux client remote controlling a Windows swarm node I guess many people still are used to unix slashes.

I also use eg. `docker run -v //./pipe/docker_engine://./pipe/docker_engine -it -p 8080:8080 stefanscherer/visualizer-windows:insider` right now and it works pretty good as Golang itself can handle backslashes and slashes in paths.

Slashes because bash needs extra escaped backslashes as you can see here.

´´´
$ docker run -v \\.\pipe\docker_engine:\\.\pipe\docker_engine -it -p 8080:8080 stefanscherer/visualizer-windows:insider
docker: Error response from daemon: invalid bind mount spec "\\.pipedocker_engine:\\.pipedocker_engine": invalid volume specification: '\.pipedocker_engine:\.pipedocker_engine'.
```


(oops as you can see even GitHub looses some backslashes although I'm using three backticks here)

Reply:
Normal slash is already handled in this function on the line above (111). A single `/` already returns true.

Reply:
Nice!